#position: [x, y, z]
#orientation: [roll, pitch, yaw]
block_printer:
  pick0:
    block: {size_x: 1,size_y: 1, color: "white"}
    position: [0.095,0.333,0.208] 
    orientation: [0,0,1.57079632679]
    capacity: 30
  pick1:
    block: {size_x: 1,size_y: 1, color: "red"}
    position: [0.095,0.327,0.208] 
    orientation: [0,0,1.57079632679]
  pick2:
    block: {size_x: 1,size_y: 1, color: "orange"}
    position: [0.095,0.321,0.208] 
    orientation: [0,0,1.57079632679]
  pick3:
    block: {size_x: 1,size_y: 1, color: "yellow"}
    #position=[1x1supplier position]+[-0.003,0.003,0.007]
    position: [0.095,0.315,0.208]
    orientation: [0,0,1.57079632679]
  pick4:
    block: {size_x: 1,size_y: 1, color: "green"}
    position: [0.095,0.309,0.208] 
    orientation: [0,0,1.57079632679]
  pick5:
    block: {size_x: 1,size_y: 1, color: "blue"}
    position: [0.095,0.303,0.208] 
    orientation: [0,0,1.57079632679]
  pick6:
    block: {size_x: 1,size_y: 1, color: "black"}
    position: [0.095,0.297,0.208] 
    orientation: [0,0,1.57079632679]
  pick7:
    block: {size_x: 1,size_y: 1, color: "support"}
    position: [0.095,0.291,0.208] 
    orientation: [0,0,1.57079632679]
  pick8:
    block: {size_x: 2,size_y: 1, color: "white"}
    position: [0.095,0.284,0.208] 
    orientation: [0,0,1.57079632679]
  pick9:
    block: {size_x: 2,size_y: 1, color: "red"}
    position: [0.095,0.278,0.208] 
    orientation: [0,0,1.57079632679]
  pick10:
    block: {size_x: 2,size_y: 1, color: "orange"}
    position: [0.095,0.272,0.208] 
    orientation: [0,0,1.57079632679]
  pick11:
    #position={supplier position}+{-0.005,0,0.007}
    block: {size_x: 2,size_y: 1, color: "yellow"}
    position: [0.095,0.266,0.208] 
    orientation: [0,0,1.57079632679]
  pick12:
    block: {size_x: 2,size_y: 1, color: "green"}
    position: [0.095,0.260,0.208] 
    orientation: [0,0,1.57079632679]
  pick13:
    block: {size_x: 2,size_y: 1, color: "blue"}
    position: [0.095,0.254,0.208] 
    orientation: [0,0,1.57079632679]
  pick14:
    block: {size_x: 2,size_y: 1, color: "black"}
    position: [0.095,0.248,0.208] 
    orientation: [0,0,1.57079632679]
  pick15:
    block: {size_x: 2,size_y: 2, color: "white"}
    position: [0.095,0.239,0.208]
    orientation: [0,0,1.57079632679]
  pick16:
    block: {size_x: 2,size_y: 2, color: "red"}
    position: [0.095,0.229,0.208]
    orientation: [0,0,1.57079632679]
  pick17:
    block: {size_x: 2,size_y: 2, color: "orange"}
    position: [0.095,0.219,0.208]
    orientation: [0,0,1.57079632679]
  pick18:
    block: {size_x: 2,size_y: 2, color: "yellow"}
    #position={supplier position}+{-0.005,0,0.007}
    position: [0.095,0.209,0.208]
    orientation: [0,0,1.57079632679]
  pick19:
    block: {size_x: 2,size_y: 2, color: "green"}
    position: [0.095,0.199,0.208]
    orientation: [0,0,1.57079632679]
  pick20:
    block: {size_x: 2,size_y: 2, color: "blue"}
    position: [0.095,0.189,0.208]
    orientation: [0,0,1.57079632679]
  pick21:
    block: {size_x: 2,size_y: 2, color: "black"}
    position: [0.095,0.179,0.208]
    orientation: [0,0,1.57079632679]
